### ISSUE TITLE
`aria-controls` attribute usage

### ISSUE THREAD

**Issue by webmandesign** [anace/keyboard-navigation#4]
## Issue description

In the JavaScript code I can see there is `aria-controls` attribute applied on the toggle button with value of `sub-menu`.

Please note that the value should be the ID of the object it controls, not the class. Also, as the sub menu is the first element following the toggle button, the `aria-controls` attribute is obsolete.

## Proposed solution

Remove `aria-controls` attribute from toggle button.

## Resources

- https://www.w3.org/TR/wai-aria-1.1/#aria-controls
- https://marcozehe.wordpress.com/2010/02/10/easy-aria-tip-5-aria-expanded-and-aria-controls/
- https://www.heydonworks.com/article/aria-controls-is-poop
